### PR TITLE
[ticket/13782] Use non-reserved word "void" for cache driver instead of null

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-13782

### CODE REVIEW

**Comment by marc1706** [comment]
If you don't like "void" we could also change it to something like "dummy".

**Comment by rxu** [comment]
BTW, we have the same issue with ```/phpbb/log/null.php```.

**Comment by s9e** [comment]
I'd be worried that `void` would become a keyword. How about `noop` for no-op. Does that make sense?

**Comment by VSEphpbb** [comment]
Other options:
`null_cache`
`is_null`
`null_void`
`non`

**Comment by marc1706** [comment]
I like the idea of calling it ```dummy``` more and more as that fits what it does pretty well.

File: phpBB/phpbb/cache/driver/void.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /**
 * ACM Null Caching
 */
-class null extends \phpbb\cache\driver\base
+class void extends \phpbb\cache\driver\base
```
Comment (Nicofuma) [inline phpBB/phpbb/cache/driver/void.php]: I think we should keep the old null class (extending the new one) for BC purpose

**Comment by Nicofuma** [comment]
I like the idea of `dummy` too

**Comment by marc1706** [comment]
Replaced by: https://github.com/phpbb/phpbb/pull/3584

**Pull request closed without merging**